### ISSUE TITLE
Transformer for Jest

### ISSUE THREAD

**Issue by segevfiner** [egoist/tsup#665]
It would be nice to have a Jest transformer that uses tsup so that we don't have to create an additional separate build configuration for Jest using babel or ts-jest.

**Comment by steveluscher** [comment]
Yes! I would _love_ this to keep the build consistent between my releases and my tests.

**Comment by MaxiSanchez600** [comment]
Any news on this?
